cmd/zed/README.md

```diff
@@ -1,7 +1,17 @@
-# `zq`
+# `zed`
 
-`zq` is a command-line tool to search, analyze, and transform structured logs.
- It evaluates [ZQL ](../../docs/language/README.md) queries against input log
+`zed` is a command-line tool to manage, search, query, and transform
+data in individual files or, with `zed lake ...`, data in a Zed lake.
+
+## Under construction
+
+This documentation is under construction.
+
+## `zq`
+
+The `zq` command is a command-line shortcut for `zed query`.
+
+`zq` evaluates [ZQL ](../../docs/language/README.md) queries against input log
```

Comment:
```suggestion
`zq` evaluates [Zed ](../../docs/language/README.md) queries (formerly known as ZQL) against input log
```

Not sure if now's the time to start phasing out "ZQL" references, but, figured I'd put it out there.

Reply:
Yes, it's time.  @nwt caught this too.